crates/bevy_dev_tools/src/hotpatch.rs

```diff
@@ -0,0 +1,38 @@
+//! Utilities for hotpatching code.
```

Comment:
I guess this would be a good place to add the hot-patch setup documentation and the limitations? Or should it go to the `Plugin`?